python/src/opendp/_convert.py

```diff
@@ -411,7 +411,7 @@ def _tuple_to_slice(val: tuple[Any, ...], type_name: Union[RuntimeType, str]) ->
     if len(inner_type_names) != len(val):
         raise TypeError("type_name members must have same length as tuple")
 
-    if inner_type_names == ['LogicalPlan', 'Expr']:
+    if inner_type_names == ['DslPlan', 'Expr']:
```

Comment:
From the Python side, a DslPlan and LazyFrame are treated interchangeably, because there's no direct representation of a DslPlan in Python. From the Rust side, LazyFrames are DslPlans with optimization settings.